README.md

```diff
@@ -152,6 +153,16 @@ You can also add your own env files other than `dev` and `prod` by doing the fol
 - add `{ NAME: 'src/environments/environment.NAME.ts' }` to the the `apps[0].environments` object in `angular-cli.json` 
 - use them by using the `--env=NAME` flag on the build/serve commands.
 
+### Base tag handling in index.html
+
+You can modify base tag (`<base href="/">`) in your index.html by using `--base-href your-url` option. It's useful when building or serving for different environments.
+
+```bash
+# Sets base tag href to /myUrl/ in your index.html
+ng build --base-href /myUrl/
+ng serve --base-href /myUrl/
```

Comment:
Does it make sense to have this flag in the serve command? I'd expect the app to not work since the Dev server isn't aware of it. Haven't tested though.